src/main/resources/templates/email/programme-created/v1.0.0.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/html">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+</head>
+<body>
+<h1>Email Message</h1>
+<h2>Subject</h2>
+<th:block th:fragment="subject">NHS England - <span
+    th:text="${not #strings.isEmpty(localOfficeName)} ? ${localOfficeName} : _" th:remove="tag"></span
+  > - Onboarding to your Training programme</th:block>
```

Comment:
We could embed one of the hyphens (and whitespace) in the conditional text to avoid having `NHS England -  - Onboarding to your Training programme" when the LO is not available.

Reply:
In the absence of anything specific, I have standardised on 'your local deanery office' wherever a missing local office name should appear.